prov/usnic/src/usdf.h

```diff
@@ -376,6 +376,7 @@ struct usdf_cq {
 	struct usdf_domain *cq_domain;
 	struct fi_cq_attr cq_attr;
 	uint8_t is_soft;
+	uint8_t waiting;
```

Comment:
I could fix waiting here, and submit a separate PR to update `is_soft` since it isn't really related to this one. That fine?